app/Code.gs

```diff
@@ -150,38 +150,43 @@ function asciidocHandleFontStyle(text, offset, distinctContent) {
   var isBold = text.isBold(offset);
   var isItalic = text.isItalic(offset);
   var isUnderline = text.isUnderline(offset);
-  var isPlain = !isBold && !isItalic && !isUnderline;
+  var isStrikethrough = text.isStrikethrough(offset);
+  // FIXME: getTextAttributeIndices doesn't split on different fonts,
+  // makeing this almost useless
   var isCode = isTextCode(text);
   // Prefix markup
   if (isUnderline) {
-    // Underline doesn't play nice with others
-    result = result + '+++<u>';
-  } else {
-    if (isBold) {
-      result = result + new Array(numOccurence).join('*');
-    }
-    if (isItalic) {
-      result = result + new Array(numOccurence).join('_');
-    }
-    if (isCode) {
-      result = result + '+';
-    }
+    result += '+++<u>+++'; // or asciidoc.css class: underline
```

Comment:
I see... I guess `+++<s>++++++<u>+++underline+++</u>++++++</s>+++` should be simplified `+++<s><u>underline</u></s>+++` but your solution now properly handle both underline and strike through :+1: